arch/Kconfig

```diff
@@ -69,15 +69,14 @@ module-str = mpu
 source "subsys/logging/Kconfig.template.log_config"
 
 config BIG_ENDIAN
-       bool
-       help
-         This option tells the build system that the target system is
-         big-endian.  Little-endian architecture is the default and
-         should leave this option unselected.  This option is selected
-         by arch/$ARCH/Kconfig, soc/**/Kconfig, or boards/**/Kconfig
-         and the user should generally avoid modifying it.  The option
-         is used to select linker script OUTPUT_FORMAT and command
-         line option for gen_isr_tables.py.
+	bool
+	help
+	  This option tells the build system that the target system is big-endian.
```

Comment:
I don't see this kind of change - re-wrapping lines to a different width - to have been described in the commit message. What's the reason for it, and how it was done? E.g., does it try to establish a consistent line width (definitely worth describing in the commit message), or did you just take a chance to do something when you fixed indent there anyway?

> (meaning another pass hopefully won't be needed)

Reply:
Used `gq` in Vim to wrap to 79 columns. Only did it for a few of them though, because I was afraid I'd mess up some formatting while batch-fixing.

Reply:
Commit message now mentions it.

Reply:
ack, thanks